### ISSUE TITLE
AVT 1 - TextArea & TextInput used in the same page have DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon-components-react#2227]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
Create a simple single page React app
Add both TextArea and TextInput components to the page
Run DAP - 3 violations reported (see screenshot)
<img width="1295" alt="Screen Shot 2019-04-17 at 8 40 57 AM" src="https://user-images.githubusercontent.com/21676914/56292989-dba43c00-60ed-11e9-940c-e2322f6910f3.png">

Note: The issue is that both components use the same id="test2" to reference the label.

**Comment by snidersd** [comment]
This issue should remain open until it is fixed and retested.

**Comment by dakahn** [comment]
The end user of these component can and should supply more specific ID's as props in implementation. 👍
